src/docs/concepts-api-sdk/rate-limiting/index.md

```diff
@@ -0,0 +1,115 @@
+environments. In addition, the Integration API listing creation endpoint
+has a separate rate limit in all environments. This means that in dev
+and demo environments, both rate limits apply to listing creation in the
+following way:
+
+- If the command rate limit burst capacity has not yet been depleted,
+  listing creation is rate limited at 100 API calls per minute. This
+  depletes the command rate limit burst capacity accordingly.
+- If the command rate limit burst capacity has been depleted, listing
+  creation is rate limited at the regular command rate limit of 30
+  requests per minute.
+- If other commands are taking place while listings are being created,
+  all those calls count towards the command rate limit.
+
+## Handling rate limits in Marketplace API
+
+In standard usage, the rate limit will largely be undetectable in
+Marketplace API. The API calls happen, for the most part, as a
+consequence of user actions, so there is a natural delay between calls.
+This means that encountering rate limit errors while developing with
+Marketplace API can point you towards finding loops and other buggy
+behavior in your code.
+
+If your test environment is exposed to the public web, a traffic spike
+caused by bots or crawlers can trigger rate-limiting, preventing
+server-side rendering from functioning. This may temporarily prevent
+access to your test environment. Therefore, we strongly recommend using
+[basic HTTP authentication](/tutorial/deploy-to-render/#enable-http-basic-access-authentication)
+on all deployed test marketplace applications.
+
+## Handling rate and concurrency limits in Integration API
+
+Rate limits will likely be more visible to you when developing with the
+Integration API. In addition, Integration API also features concurrency
+limiting, which needs to be handled.
+
+### Integration SDK
+
+Flex Integration SDK is configured to handle concurrency limits by
+default starting from version XX-XX. To take this built-in concurrency
+limiting to use, you need to make sure your SDK is updated to at least
+this version.
+
+To handle the general rate limits in dev and demo environments, you can
+pass configurations for query and command rate limiters. You can see the
+details of passing those configurations in our
+[SDK documentation](TODO).
+
+For listing creation rate limits, you will need to implement your own
+rate limit handling logic. We have an example of this in our
+[Integration API example scripts repository](https://github.com/sharetribe/flex-integration-api-examples/blob/master/scripts/create-listings.js).
+
+It is good to note that rate limits apply by client IP address. If you
+have more than one instance of the SDK running on the same server or
+computer, then each SDK instance will rate-limit itself, but combined
+they might still go over the total limits. If that is the case, you can:
+
+- customize the rate limits to lower values than the suggested ones,
```

Comment:
We could add link to the SDK docs about this. The link will be: `https://sharetribe.github.io/flex-integration-sdk-js/rate-limits.html` but that's a 404 for now, as I have not published the docs yet.